doc/user/setup.rst

```diff
@@ -153,3 +154,78 @@ location. A good place is usually ``/etc/systemd/system/``.
 After issuing a ``systemctl daemon-reload``, you should be able to start the
 FRR service via ``systemctl start frr``. If this fails, or no daemons are
 started. check the ``journalctl`` logs for an indication of what went wrong.
+
+Operations
+----------
+
+This section covers a few common operational tasks and how to perform them.
+
+Restarting
+^^^^^^^^^^
+
+Restarting kills all running FRR daemons and starts them again. Any unsaved
+configuration will be lost.
+
+Without systemd
+"""""""""""""""
+
+::
+
+   /usr/lib/frr/frrinit.sh restart
+
+or::
+
+   service frr restart
```

Comment:
`service frr reload` works in all cases, if systemd is in use it just calls `systemctl reload`

Also, `frrinit.sh` is symlinked to `/etc/init.d/frr`, which is probably better to reference here.  Main reason for putting it in `/usr/lib/frr` is that the systemd service file uses it too, and it'd be pretty weird to call into `/etc/init.d` from a service file.

(I'd probably mention `service` first, and `systemctl`/`/etc/init.d` only as sidenotes.)

Reply:
I am aware that `service blah` is a systemd shim on systemd systems.

```
(I'd probably mention service first, and systemctl//etc/init.d only as sidenotes.)
```

Fine by me